nav2_dwb_controller/dwb_core/package.xml

```diff
@@ -46,5 +48,6 @@
 
   <export>
     <build_type>ament_cmake</build_type>
+    <LocalPlanner plugin="${prefix}/dwb_core.xml" />
```

Comment:
I'm pretty sure this is supposed to be `nav2_core`

Reply:
@SteveMacenski The ```dwb_core```.xml name is not the most descriptive. This xml is basically to make the plugin available to ROS toolchain. It has the plugin library path, type and base class information:
```  
<library path="dwb_core">
    <class type="dwb_core::DWBLocalPlanner" base_class_type="nav2_core::LocalPlanner">
 ```
And this is being exported here in the ```package.xml```.
It was earlier named as just ```plugin.xml```.
You have named it like ```global_planner_plugin.xml```, so similarly I can change the name to ```local_planner_plugin.xml```, rather than ```nav2_core```.

You have already approved the changes, and Carl has too, so we can go ahead and merge it.
I will make the name change and submit a PR.

Reply:
I'm not talking about the `xml` name, the export target is supposed to be the pkg, ei `LocalPlanner` is supposed to be `nav2_core`, if you also look at my planner work & other pluginlib packges like this: https://github.com/vinnamkim/teb_local_planner/blob/kinetic-devel/package.xml#L48

This is for tools to register the plugins for search, I'm actually really surprised it works for you with that incorrect but it also may be because ament/colcon doesnt have any cli tools for introspection yet

Reply:
Got it, thanks.
I will change it.